spec/requests/submissions_spec.rb

```diff
@@ -28,10 +29,28 @@
       end
 
       it "sends the claim's details to the “submitted” dataset on Geckoboard" do
+        perform_enqueued_jobs { post claim_submission_path(StudentLoans.routing_name) }
         expect(@dataset_post_stub.with { |request|
           request_body_matches_geckoboard_data_for_claims?(request, [in_progress_claim.reload])
         }).to have_been_requested
       end
+
+      # None of these specs should be here, should be in features.
```

Comment:
@nomasprime you mean the examples above? All of them? Lets pick up in tech debt phase after Private Beta

Reply:
@nomasprime Also interesting is the existing specs only concentrate on TSLR (StudentLoans) not M+P (MathsAndPhysics), and what we are adding EarlyCareerPayments (ECP). 

Do you envisage we need to update this for ECP?

Or follow the regular pattern I have observed where only one Policy is generally concentrated on?

Reply:
Specs generally should be refactored and should test what should be tested.